arrays-alex-dane.js

```diff
@@ -0,0 +1,127 @@
+console.log(slicedarr)
+
+// Consider the variable:
+
+var numbers = [2, 4, 6, 8, 10]
+// Write the code that will add the number 0 to the beginning of the array.
+
+console.log(numbers.unshift(0))
+
+// Write the code that will add the number 12 to the end of the array.
+var numbers = [2, 4, 6, 8, 10]
+console.log(numbers.push(12))
+console.log(numbers)
+
+// Write the code that will remove the first number from the array.
+var numbers = [2, 4, 6, 8, 10]
+console.log(numbers.shift(0))
+console.log(numbers)
+
+// Write the code that will add the number 0 to the beginning of the array without altering the original array. HINT: it's not .unshift You'll have to get creative! ;)
+var numbers = [2, 4, 6, 8, 10]
+var numbers2 = [0]
+console.log(numbers2.concat(numbers))
+console.log(numbers)
+
+
+// Consider the variable:
+
+var numSet = [2, 13, 6, 8, 4, 2]
+// Write the code that finds the index of the first appearance of the number 2.
+
+console.log(numSet.indexOf(2))
+
+// Write the code that finds the index of the last appearance of the number 2.
+
+console.log(numSet.lastIndexOf(2))
+
+// Write the code that returns the number at the third index.
+
+console.log(numSet[3])
+
+// Consider the variable:
+
+var characters = ["y", "a", "r", "r", "a"]
+// Write the code that brings all the letters in the characters array together into a string.
+
+console.log(characters.join())
+
+// Write the code that reverses the order of the letters in the characters array and saves it into a variable called charsReversed.
+
+var charsReversed = characters.reverse()
+console.log(charsReversed)
+
+// Write the code that brings all the letters in the charsReversed array together into a string with an asterisk between each letter.
+
+console.log(charsReversed.join("*"))
+
+// Write the code that brings all the letters in the charsReversed array together into a string without separators.
+
+console.log(charsReversed.join(""))
+
+// Create two arrays consisting of three first names of your cohort members in each.
+
+var arr1 = ["Alex", "Dane", "Nate"]
+var arr2 = ["Stevie", "Kendra", "Paula"]
+
+// Write the code that sorts the names in alphabetical order.
+
+var sorted1 = arr1.sort()
+var sorted2 = arr2.sort()
+console.log(sorted1)
+console.log(sorted2)
+
+// Write the code that sorts the names in reverse alphabetical order.
+
+console.log(sorted1.reverse())
+console.log(sorted2.reverse())
+
+// Write the code that sorts all the names in alphabetical order in a single array.
+
+var sortedNames = arr1.concat(arr2).sort()
+console.log(sortedNames)
+
+// Consider the variables:
+
+var numbers = [42, 221, 71, 7, 18, 87]
+var oddIndexes = []
+// Write the code that logs the values from the numbers array that are at odd indexes.
+
+for (let i = 0; i < numbers.length; i++) {
+    var current = numbers[i]
+    if (current % 2 !== 0) {
+        console.log(current)
+    }
+}
+
+// Write the code that adds the values from odd indexes into the oddIndexes array.
```

Comment:
Similar from above. Refactor this for loop to begin at the first odd index and increment to the next odd index. Your push method is spot on!